Tests/test_exceptionconverter.py

```diff
@@ -1,142 +0,0 @@
-#####################################################################################
```

Comment:
No tests were being run from this file and the framework for running the test no longer exists in the testing libraries.